docs/api/parks-post-parks.md

```diff
@@ -1,17 +1,59 @@
-# POST Parks
+# Add a New Park
 
-### URL
+## `POST` /parks
 
 ```
 {server_url}/parks
 ```
 
-### Parameters
+Create a new park object.
 
-### Request headers
+### Headers
 
-### Request body
+None
 
-### Return body
+### Path Parameters
 
-### Return status
+None
+
+### Query Parameters
+
+None
+
+### Request Body
+
+| Name | Data Type | Required/Optional | Description | Example |
+| --- | --- | --- | --- | --- |
+| park_name | String | Required | The name of the park | Yosemite National Park |
+| location | String | Required | The city or region of the park | Sierra Nevada Mountains |
+| habitat | String | Required | A brief description of the environment | mountains |
+| fun fact | String | Required | A brief fact that makes the park unique | Yosemite Falls, the tallest waterfall in North America, is actually composed of three falls: Upper Yosemite Fall, the middle cascades, and Lower Yosemite Fall.  |
```

Comment:
Maybe we could update this to "A brief fact about what makes the park unique" for accuracy/clarity.